src/Avalonia.Controls/Primitives/ToggleButton.cs

```diff
@@ -182,13 +211,13 @@ private void OnIsCheckedChanged(AvaloniaPropertyChangedEventArgs e)
             switch (newValue)
             {
                 case true:
-                    OnChecked(new RoutedEventArgs(CheckedEvent));
+                    OnChecked(new RoutedEventArgs(IsCheckedChangedEvent));
```

Comment:
There is an open question of whether the OnChecked, OnUnchecked and OnIndeterminate protected virtual methods should also be removed in favor of a single OnIsCheckedChanged method. This would break those that inherit from this control. However, these methods are not used anywhere else in Avalonia's codebase.

Reply:
I forgot I still need to raise the old events for now. Therefore, I do need to add a separate `OnIsCheckedChanged` method.